src/main.rs

```diff
@@ -92,7 +101,19 @@ fn get_json(extra_args: &[&str]) -> Result<String, ProgramError> {
     Ok(String::from_utf8(output.stdout)?)
 }
 
-fn handle_suggestions(suggestions: &[Suggestion]) -> Result<(), ProgramError> {
+#[derive(PartialEq, Eq, Debug)]
+enum AutofixMode {
+    /// Do not apply any fixes automatically
+    None,
```

Comment:
But… but… it took me _minutes_ to come up with `BetterSafeThanSorry` 😿

Okay… fine. Have it your way.